ssz-rs/src/lib.rs

```diff
@@ -9,21 +14,19 @@ mod merkleization;
 mod ser;
 #[cfg(feature = "serde")]
 mod serde_test;
+mod std;
 mod uint;
 mod union;
 mod vector;
 
-use crate::list::Error as ListError;
-use crate::vector::Error as VectorError;
+pub use crate::std::*;
```

Comment:
does `ssz-rs` need to expose `crate::std::*`?

it seems like no but perhaps we need to export our exact types for other no-std code to use ours?

Reply:
We don't need to expose it, you're right. Addressed in https://github.com/ralexstokes/ssz-rs/pull/25/commits/02961035ee736414a965e9f8fefceae01e1c98a4.